specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -1815,7 +1826,7 @@
           "HostPoolUpdate"
         ],
         "operationId": "HostPoolPost_Update",
```

Comment:
why do we have /updateDetails for GET and not just having GET on /update?

Reply:
/update an action.
/updateDetails is the proxy object that returns the state of the hostpool update that is a result of the /update action.

Reply:
I think i must be missing something about the rest api design.  I though the POST would be the action part, and GET would retrieve the object that POST created, at the same endpoint.

Reply:
No post can have a different action Name. look at sendMessage for userSession.